CHANGELOG.md

```diff
@@ -17,6 +17,8 @@
 - Feedbacks functions to set/get feedback polarity
 - Add in configuration set_velocity_pid, set_position_pid and get_rated_current
 - Add functions to connect to and scan EtherCAT devices using CoE (SDOs).
+- Running Smoke and All tests stages for the Jenkins pipeline.
+- Option to choose running only smoke and no connection tests, or all tests.
```

Comment:
I don't think these changes should be in the changelog. They are not library changes, they are changes in the process.
What do you think?

Reply:
But this PR adds the option to choose what the developer wants. I had to add a parameter for that in the Jenkinsfile. And the Jenkinsfile is inside the library, isn't it?

Reply:
It refers to the development, not to the use.

Reply:
I also think that this change should not be in the CHANGELOG